CIDER/cider.py

```diff
@@ -42,14 +44,19 @@
 
 import chemplot as cp
 from chemplot import descriptors
+from bokeh.plotting import output_file, save
 
 from typing import List, Tuple, Dict
 from itertools import count
+import PIL
+import matplotlib
 
 import logging
+import sys
 
-logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('CIDER')  # needs to be here
```

Comment:
Could you configure the logger here as we discussed and remove the example file "test_logging.py"?

Reply:
Ah, it is below the constructor, right? I would move it up and put it after "class ChemicalDatasetComparator:..."

Reply:
I was confused here because you define the variable logger two times with the same value.

Reply:
The definition 'logger=...' needs to be outside of the class. If I move it inside the class (where the basic configuration of the logger is), I get warnings that logger is not defined and when I try to run the test function it will fail due to this. Therefore, I left the definition of the logger where it was and added the commend. And it seems I forgot to delete the second definition in the class.

Reply:
Is it possible to move the whole logging setup outside of the class, then?

Reply:
Yes, this is as it was before and that works.

Reply:
Good, it would be nice to have that all in one place together.